src/client/src/routes/StyleguideRoute/elements.tsx

```diff
@@ -0,0 +1,97 @@
+import _ from 'lodash'
+import React from 'react'
+import { styled, Styled } from 'rt-theme'
+
+import { resolvesColor } from 'rt-theme'
+
+import { mapMarginPaddingProps, mapTextProps, MarginPaddingProps, TextProps } from './styled'
+
+export { Paragraph } from './styled'
+
+export const Hashable = ({ is: Element = 'div', ...props }) => <Element {...props} id={_.kebabCase(props.children)} />
+
+interface HeaderProps extends MarginPaddingProps, TextProps {}
+export const Header: Styled<HeaderProps> = styled(Hashable)`
```

Comment:
General comment for all the places you have this:

`Component: Styled<Props>` --> `Component = styled<Props>('div')`

You shouldn't need the `Styled` interface.

Reply:
It continues to be a useful tool — at times typescript objects to that method. Additionally the typescript + styled tooling is incomplete and features are lost by that approach. Correction and formatting on CSS is lost, for example. And, as mentioned before, it is actually less consistent without how most types are described.

Reply:
How would you achieve typing on the `is` prop — a convention from `styled–system`:
https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/854#diff-612387fc2db9d8ac0322d7125d4ea86cR137

Reply:
Leave a comment on the exports of where you create the `Styled` type to explain why you're using them.